js-contribution-guidelines.md

```diff
@@ -0,0 +1,12 @@
+# JS Contribution Guidelines
+
+Many IPFS projects use javascript. Please check these guidelines before contributing js code to an IPFS repository.
+
+## Suggestions
+
+- Build system: Use `npm run` for small projects; for larger ones, use `gulp`.
+- Testing node: `mocha`
+- Testing browser: `karma` + `mocha`
+- Browser building: `webpack` or `browserify` (this fight is still ongoing ;)
```

Comment:
I still suggest we put `webpack` here as the main choice as we are using that atm in all new projects. These are only guidelines no hard rules anyway.